docs/cn/installation/start-with-k8s.md

```diff
@@ -202,6 +202,19 @@ spec:
 wget https://raw.githubusercontent.com/alibaba/ilogtail/main/example_config/start_with_k8s/ilogtail-daemonset.yaml
 ```
 
+添加参数示例
+```
+        - name: logtail
+          command:
+          - /usr/local/ilogtail/ilogtail_control.sh
+          - start_and_block
```

Comment:
Did you test it?
Isn't start_and_block an arg itself?

Reply:
> Did you test it? Isn't start_and_block an arg itself?

Yes, I wrote it in the wrong place.